pgd_core/fixtures/new_users.json

```diff
@@ -0,0 +1,38 @@
+[
```

Comment:
The three user fixtures in this directory should be combined, and renamed 'pgd_core.json'.  All the test data for the entire app should be in this one fixture.